RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -991,7 +991,9 @@ public enum FeePayerValidationOutcome: Sendable, Hashable {
 
 extension ReviewedTransaction {
 	var involvedAccounts: Set<AccountAddress> {
-		Set(accountWithdraws.keys).union(accountDeposits.keys)
```

Comment:
For manifests without any withdraws or deposits, the involvedAccounts would have been empty, thus any fee payer would have been considered as a new account for this transaction, triggering the warning to be shown.